### ISSUE TITLE
[BUG] Files in the research folders display as links

### ISSUE THREAD

**Issue by yoda-eur** [UtrechtUniversity/yoda-ansible#372]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

The files in the research folders are now displayed as links and are clickable, leading to an error when clicked.

### Expected Behavior

Should not be clickable

### Steps To Reproduce

On the Yoda website, clicking a file in one of the research folders.

### Environment

```markdown
- Yoda: v1.9
- Ansible:?
- Operating System: Windows
- Browser: every browser
```

### Anything else?

<img width="500" alt="files-show-as-link-and-are-clickable" src="https://github.com/UtrechtUniversity/yoda/assets/59565301/270dfb86-428c-4370-8ab5-0480856dd3cf">
<img width="737" alt="result-after-clicking-file" src="https://github.com/UtrechtUniversity/yoda/assets/59565301/347979fa-132f-4ec3-9d8e-274053e92295">

**Comment by Danny-dK** [comment]
Note that clickable files is not the bug here. The Yoda devs have worked to make certain files be viewable in the web portal. These files usually are image, video, or text based files (this was a longstanding feature request). Therefore, files that are viewable are clickable. Some files are just too large to be viewed in the webportal (leading to that message) or some file that are not large do still elicit an error (meaning a possible bug in this new file viewer).

**Comment by fiberbit** [comment]
Another possibility might be that your user is of "rodsadmin" type? These users "see" all files and folders in listings but certainly do not have automatic "read" permission on the *contents* of data objects, unless they were added specifically. The error message in that case is a bit confusing perhaps.

**Comment by lwesterhof** [comment]
I have just confirmed with SURF that this instance is still on rc.6. This will be fixed when upgraded to v1.9.0.

**Comment by fiberbit** [comment]
Just a small note that as a rodsadmin user I see exactly this behaviour still on v1.9.0 but hopefully for "normal" users it is fixed indeed.

**Comment by yoda-eur** [comment]
Thank you all for replying.  

@Lazlo, good to know this will be fixed in v1.9.0. I was testing in the acceptance 1.9 environment of Yoda. That one is not showing Yoda v1.9.0-rc.6, like it do in our production environment.
